pom.xml

```diff
@@ -76,5 +76,10 @@ THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
             <version>1.10.19</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.google.guava</groupId>
```

Comment:
I guess "contract" was incorrect - but where I saw it being used in the dropwizard library they were constructing unmodifiable maps.

Ok I'll take the feedback.